epochStart/bootstrap/epochStartMetaBlockProcessor.go

```diff
@@ -191,6 +192,27 @@ func (e *epochStartMetaBlockProcessor) GetEpochStartMetaBlock(ctx context.Contex
 	}
 }
 
+func (e *epochStartMetaBlockProcessor) getMostReceivedMetaBlock() (*block.MetaBlock, error) {
+	e.mutReceivedMetaBlocks.RLock()
+	defer e.mutReceivedMetaBlocks.RUnlock()
+
+	const hashNotAvailable = "N/A"
+	mostReceivedHash := hashNotAvailable
+	maxLength := minNumOfPeersToConsiderBlockValid - 1
+	for hash, entry := range e.mapMetaBlocksFromPeers {
```

Comment:
Could it happen that the mapMetaBlocksFromPeers contains multiple entries from the same peer? Does it somehow influnece the selected metablock if we just have the minimum number.

Reply:
no, there is a check when entries are added to that map so that the slice of an entry doesn't contain duplicate peers

Reply:
ok, clarified that the slice for the same hash does not contain dupplicates, but for different hashes we could have the same peer.